testsuite/openshift/types/__init__.py

```diff
@@ -22,8 +22,15 @@ def fetch_resource(self, name, *cmd_args: str, auto_raise: bool = True):
 
     def __iter__(self):
         """Return iterator for requested resource"""
-        data = self._client.do_action("get", self._resource_name, parse_output=True)
-        return iter(data["items"])
+        data = self._client.do_action(
+            "get",
+            self._resource_name,
+            "-o",
+            "json",
```

Comment:
Why do you need it in json, it can be parsed in the yaml as well

Reply:
It doesn't matter in the context. Anyway in the next line output will be converted to a python dictionary.